test/core/event_engine/test_suite/iomgr_event_engine_test.cc

```diff
@@ -20,10 +20,12 @@
 int main(int argc, char** argv) {
   testing::InitGoogleTest(&argc, argv);
   grpc::testing::TestEnvironment env(&argc, argv);
-  SetEventEngineFactory([]() {
-    return absl::make_unique<
-        grpc_event_engine::experimental::IomgrEventEngine>();
-  });
+  SetEventEngineFactories(
+      []() {
+        return absl::make_unique<
+            grpc_event_engine::experimental::IomgrEventEngine>();
+      },
+      nullptr);
```

Comment:
This should either be the posix oracle or the IomgrEventEngine itself, the former preferred.

Reply:
The posix oracle can only build on linux/mac. When we reach the stage of testing endpoints, I was planning to create platform specific versions of this file.